### DISCUSSION TITLE
Disable cache via Environment variable

### DISCUSSION THREAD

**Discussion by EWhite613** [vercel/turborepo#7819] [Help] [answered]
### Goals

1. Want to disable caching via an Environment variable so can force CI runs to not use cache

### Background

NX has support for this via `NX_SKIP_NX_CACHE`

### Proposal

Create an environment variable for an equivalent of `--no-cache`. Something like `TURBO_NO_CACHE`, if set will not use cache

**Comment by anthonyshew** [comment, marked as answer]
`TURBO_FORCE`: https://turbo.build/repo/docs/reference/system-variables
